src/ripple/app/misc/NetworkOPs.h

```diff
@@ -109,7 +109,7 @@ class NetworkOPs
     // Use *only* to timestamp our own validation
     virtual std::uint32_t getValidationTimeNC () = 0;
     virtual void closeTimeOffset (int) = 0;
-    virtual boost::posix_time::ptime getNetworkTimePT () const = 0;
+    virtual boost::posix_time::ptime getNetworkTimePT (int& offset) const = 0;
```

Comment:
The returned integer is some kind of seconds I think?  Sure would be nice if there were a comment to describe that.  I also think the `PT` stands for Posix Time?  Maybe?  I'm not at all sure what the `NC` means on the other methods, but that pre-dates your change.